src/main/java/blusunrize/immersiveengineering/common/util/compat/crafttweaker/CrTIngredientUtil.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * BluSunrize
+ * Copyright (c) 2020
+ *
+ * This code is licensed under "Blu's License of Common Sense"
+ * Details can be found in the license file in the root folder of this project
+ */
+package blusunrize.immersiveengineering.common.util.compat.crafttweaker;
+
+import blusunrize.immersiveengineering.api.crafting.FluidTagInput;
+import blusunrize.immersiveengineering.api.crafting.IngredientWithSize;
+import blusunrize.immersiveengineering.api.crafting.StackWithChance;
+import com.blamejared.crafttweaker.api.item.IIngredient;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.impl.item.MCWeightedItemStack;
+import com.blamejared.crafttweaker.impl.tag.MCTag;
+import net.minecraft.fluid.Fluid;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.Ingredient;
+import net.minecraft.tags.ITag;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.math.MathHelper;
+
+public class CrTIngredientUtil
+{
+
+	private CrTIngredientUtil()
+	{
+	}
+
+	/**
+	 * So far, only IItemStack supports setting an amount.
+	 * So we can at least check for that, I guess?
+	 */
+	public static IngredientWithSize getIngredientWithSize(IIngredient crafttweakerIngredient)
+	{
+		final Ingredient basePredicate = crafttweakerIngredient.asVanillaIngredient();
+		if(crafttweakerIngredient instanceof IItemStack)
+		{
+			return IngredientWithSize.of(((IItemStack)crafttweakerIngredient).getInternal());
+		}
```

Comment:
Formatting (braces should be omitted for single-statement blocks). This is consistent in the entire diff, so I won't comment on other cases.

Reply:
Applied the codestyle, so this should all be good now ^^